File: Funciones/18. Tabla_5_while/js/tabla.js
/**
 * Tabla de 5 con While
 * Autor: Santiago GM
 * Fecha: 03/04/2024
 */

let tablas;


function tabla(ptablas){

    tablas = ptablas;

    let contador = 0;
    let limite = 5;
    let resultado;
    let msg = "";

    while (contador < limite) {

        contador = contador +1;
        resultado = tablas * contador; 

        msg += `${tablas} x ${contador} = ${resultado}\n` 
    }

    return msg;
}

File: python/18. Tabla_5_while/utils/tabla.py
#Tabla de 5 con While
#Autor: Santiago GM

#Como Parametro

def tabla(ptablas,plimite):

    tablas = ptablas
    limite = plimite

    contador = 0
    resultado = 0
    msg = ""

    while (contador < limite) :

        contador = contador +1
        resultado = tablas * contador

        msg += f'{tablas} x {contador} = {resultado}\n'
    return msg
